docs/release-notes/cloud/cloud-2022-08.md

```diff
@@ -0,0 +1,101 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud August 2022.
+included_jira_versions: ['2022.Q3.4', '2022.Q3.5', '2022.Q3.6']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.2.24
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.2.59
+---
+
+# Cloud August 2022
+
+These release notes are for the Codacy Cloud updates during August 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The [GitHub integration](../../repositories-configure/integrations/github-integration.md) now uses a GitHub Apps token instead of a personal one so that the following features continue working even if the person who created the integration leaves the Codacy organization:
```

Comment:
```suggestion
-   The [GitHub integration](../../repositories-configure/integrations/github-integration.md) now uses a GitHub Apps token instead of a personal one so that the following features continue working even if the person who created the integration leaves the organization on GitHub:
```

Reply:
Excellent suggestion @lventura-codacy, this small change helps to clarify the behavior from a more user-centered standpoint. :raised_hands: